skimage/measure/fit.py

```diff
@@ -765,7 +767,8 @@ def ransac(data, model_class, min_samples, residual_threshold,
     >>> dst[0] = (10000, 10000)
     >>> dst[1] = (-100, 100)
     >>> dst[2] = (50, 50)
-    >>> model, inliers = ransac((src, dst), SimilarityTransform, 2, 10)
+    >>> model, inliers = ransac((src, dst), SimilarityTransform, 0.5, 10,
```

Comment:
You should create a new example for new functionality.

It seems that `min_samples=2` would return the same thing in `0.14.1` as it would when your patch comes in. Changing the examples makes it seem like users need to change from 2, to 0.5

Reply:
I do not understand how set `min_samples` to 2 is the same as using 50% of 50 points?

Reply:
I was confused about that too. So in the usecase that you showed, `min_samples=2` and `min_samples_fraction=0.5` with `init_inliners=np.ones(....` gives the same result? 

That definitely warrants an *additional* example (even if it is just a partial example)

Reply:
no, `initial_inliers` is completely different use-case then `min_samples`
I am against introducing new `min_samples_fraction` which is duplicated to the `min_samples`, it would become confusing for users what the primary and/or secondary parameter

Reply:
I completely agree that it is a different usecase.

As it is currently, you have erased an old use-case from the examples. 
The author of that example thought that it was worthwhile to include it.

I think you can either:
1. Show a new example with the added parameters.
2. Explain how in this case, `min_samples=2` and (`min_samples_fraction=0.5, init_inliers=np.ones(...`) give the same result.

it is like changing between wavelength, optical frequency, optical frequency in radians depending on some arbitrary threshold. It is really useful for a particular restrictive usecase, but it gets very confusing when you have to accomodate many different users.

Reply:
it is just combining case in one example...

Reply:
Ok. If you think that is best.

Reply:
I think that these two cases are not in contradiction and they are still clear...

Reply:
@Borda this line still needs removing.